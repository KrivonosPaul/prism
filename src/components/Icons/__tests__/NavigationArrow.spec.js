import React from "react";
import renderer from "react-test-renderer";

import { NavigationArrowIcon } from "../../../";

describe("NavigationArrowIcon", () => {
  it("renders correctly", () => {
    const component = renderer.create(
      <NavigationArrowIcon size="large" color="#000" />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
