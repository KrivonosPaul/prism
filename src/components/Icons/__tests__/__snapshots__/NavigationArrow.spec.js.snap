// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationArrowIcon renders correctly 1`] = `
<svg
  height="24"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <g>
    <circle
      cx="12"
      cy="12"
      fill="none"
      r="11.5"
      stroke="#000"
      strokeOpacity=".5"
    />
    <path
      d="M 7.612 12.735 L 17.878 5.935 L 11.922 16.902 L 11.755 12.68 L 7.612 12.735 Z"
      fill="none"
      stroke="#000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeOpacity=".5"
    />
  </g>
</svg>
`;
